api/v1/stats/publications/PKPStatsPublicationHandler.php

```diff
@@ -727,6 +755,8 @@ protected function _processAllowedParams(array $requestParams, array $allowedPar
                     }
                     $returnParams[$requestParam] = array_map('intval', $value);
                     break;
+                default:
+                    $this->_processAppSpecificAllowedParams($requestParam, $value, $returnParams);
```

Comment:
This is another place where you could probably avoid using an extra function. Instead, you can extend the existing function in a child class:

```php
// APP\api\v1\stats\publications\StatsPublicationHandler
protected function _processAllowedParams(array $requestParams, ...) {
    $returnParams = parent::_processAllowedParams(...);
    if ($requestParams['issueIds']) {
        $returnParams['issueIds'] = ...;
    }
    return $returnParams;
}
```